app/code/Magento/MediaGallery/Model/Keyword/Command/GetAssetKeywords.php

```diff
@@ -31,27 +32,35 @@ class GetAssetKeywords implements GetAssetKeywordsInterface
      */
     private $assetKeywordFactory;
 
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
+
     /**
      * GetAssetKeywords constructor.
      *
      * @param ResourceConnection $resourceConnection
      * @param KeywordInterfaceFactory $assetKeywordFactory
+     * @param LoggerInterface $logger
      */
     public function __construct(
         ResourceConnection $resourceConnection,
-        KeywordInterfaceFactory $assetKeywordFactory
+        KeywordInterfaceFactory $assetKeywordFactory,
+        LoggerInterface $logger
```

Comment:
The new constructor parameters should be optional in order to avoid backward compatibility issues.
Please check the [Contribution Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/#adding-a-constructor-parameter) (section *Adding a constructor parameter*) for more details.

Reply:
@dmytro-ch thank you for your review.

Correct me if I am wrong: this class is not an `@api`, and we deliver this functionality only to the developing state of the Magento `2.3-develop` As I remember, those classes will be an `@api` at the next release or something similar. So, maybe there is no need to make a suggested change because there is no backward compatibility.

Reply:
@coderimus, you're right, MediaGallery has not been included in the latest release yet. I missed this point. :)
Thank you for clarification.